extensions/amp-story/1.0/amp-story-draggable-drawer.js

```diff
@@ -85,18 +85,21 @@ export class DraggableDrawer extends AMP.BaseElement {
     /** @protected {?Element} */
     this.contentEl_ = null;
 
+    /** @private {?number} Max value in pixels that can be dragged when opening the drawer. */
+    this.dragCap_ = null;
```

Comment:
WDYT about making this non-nullable and initializing it to `Infinity` to remove the conditionals below?